css-three-column-layout/styles.css

```diff
@@ -0,0 +1,65 @@
+* {
+  box-sizing: border-box;
+  background: #3396da;
+  font-family: Lato, sans-serif;
+}
+
+.container {
+  padding-top: 100px;
+  padding-right: 200px;
+  padding-left: 200px;
```

Comment:
Is this just to center the container? I would set a width, and apply `margin: auto` to center it out. In general, you want to avoid large values for margin and padding.